app/forms/support_interface/application_forms/edit_address_details_form.rb

```diff
@@ -34,36 +36,25 @@ def self.build_from_application_form(application_form)
       def save_address(application_form)
         return false unless valid?(:address)
 
-        if uk?
-          application_form.update!(
-            address_line1: address_line1,
-            address_line2: address_line2,
-            address_line3: address_line3,
-            address_line4: address_line4,
-            postcode: postcode&.upcase,
-            country: 'GB',
-            international_address: nil,
-            audit_comment: audit_comment,
-          )
-        else
-          application_form.update(
-            address_line1: nil,
-            address_line2: nil,
-            address_line3: nil,
-            address_line4: nil,
-            postcode: nil,
-            international_address: international_address,
-            audit_comment: audit_comment,
-          )
-        end
+        attrs = {
+          address_line1: address_line1,
+          address_line2: address_line2,
+          address_line3: address_line3,
+          address_line4: address_line4,
+          postcode: postcode&.upcase,
+          international_address: nil,
```

Comment:
I'm wondering whether we should avoid overwriting this automatically so that we don't have to worry about what happens between the manual updates and activating the feature flag. I think this column can be dropped altogether when the dust settles.

Reply:
I had the same thought but decided that since it's captured in the audit log that wouldn't be necessary.